libr/bin/format/pe/pe.c

```diff
@@ -3372,10 +3372,16 @@ struct r_bin_pe_section_t* PE_(r_bin_pe_get_sections)(struct PE_(r_bin_pe_obj_t)
 		sections[j].size = shdr[i].SizeOfRawData;
 		sections[j].vsize = shdr[i].Misc.VirtualSize;
 		if (bin->optional_header) {
-			int sa = R_MAX (bin->optional_header->SectionAlignment, 0x1000);
-			ut64 diff = sections[j].vsize % sa;
-			if (diff) {
-				sections[j].vsize += sa - diff;
+			ut32 sa = bin->optional_header->SectionAlignment;
```

Comment:
this sectionalignemnt value can be fuzzed, and i think the pe loader will ignore it in case of being invalid. this probably needs some testing, but i think it will break some messed up binaries. i dont have windows to test here

Reply:
Travis is green for now 👍